skale-nodes/ansible/run_monitor.sh

```diff
@@ -0,0 +1,10 @@
+cd ../..
+source venv/bin/activate
+export BASE_DIR=skale-nodes/ansible/files
+export INVENTORY_DIR=skale-nodes/ansible/inventory
+echo 'Generating hosts'
+python skale-nodes/ansible/utils/generate_hosts.py
```

Comment:
I think it will be useful to split this script into two smaller ones. First script will create hosts file. And the second will run monitor script.

Reply:
done